### ISSUE TITLE
Loading Crash

### ISSUE THREAD

**Issue by SnowJove** [TheLimePixel/Modern-Skyblock-3#524]
Everytime I load 3.3.1.1 - 3.3.1.2 -3.3.1.3 or 3.4 i get a crash 
https://pastebin.com/S9kAKzPL

**Comment by TheLimePixel** [comment]
Is this when loading a world or your game?

**Comment by SnowJove** [comment]
No, i when when i'm loading the game about half way on the 6/7 loading it crashes
I cant even open the game

**Comment by samiratou** [comment]
I have had the same issue, intermittently, in 3.4.1.2. The pack loaded fine this morning, played for a bit, quit out of my world and the pack, went to lunch, came back & tried to fire the pack up. I'm not sure how far it got, but it didn't make it to the loading screen.

Loading the pack through MultiMC with 8GB allocated. Windows 10.

https://paste.ee/p/TD3fE
